src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -735,7 +737,52 @@ def minimize_crash(self,
                      timeout,
                      artifact_prefix=None,
                      additional_args=None):
-    return new_process.ProcessResult()
+    self._test_ssh()
+
+    additional_args = copy.copy(additional_args)
+    if additional_args is None:
+      additional_args = []
+    additional_args.append(constants.MINIMIZE_CRASH_ARGUMENT)
+    # Similarly to Linux: we do timeout / 2 here, because libFuzzer uses
+    # max_total_time for ndividual runs of the target and not for the entire
+    # minimization.
+    # Internally, libFuzzer does 2 runs of the target every iteration. This is
+    # the minimum for any results to be written at all.
+    max_total_time = (timeout - self.LIBFUZZER_CLEAN_EXIT_TIME) // 2
```

Comment:
can you please move this into a function (e.g. `get_minimize_total_time`, which takes in the given `timeout` and does this?) Then we can share the code in both places.